tests/testthat/test-child-model.R

```diff
@@ -76,7 +76,8 @@ test_that("CLHIV align", {
   out <- run_model(demp, parameters, 1970:2030, NULL, run_child_model = TRUE)
 
   spec_prev <- input$offtrt
-  hc1 <- dplyr::right_join(data.frame(reshape2::melt(out$hc1_hiv_pop)), data.frame(Var1 = 1:7, cd4_cat = c("gte30", "26-30", "21-25", "16-20", "11-5", "5-10", "lte5")), by = "Var1")
+
+  hc1 <- dplyr::right_join((reshape2::melt(out$hc1_hiv_pop)), data.frame(Var1 = 1:7, cd4_cat = c("gte30", "26-30", "21-25", "16-20", "11-5", "5-10", "lte5")), by = "Var1")
```

Comment:
Typo here: `11-5` -> `11-15`.

Maybe suffix these with `perc` for clarity: `26-30perc`

Reply:
Fixed the type, not how much adding perc would actually clarify so kicking that down the road